config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
 
-  get 'news_feed/index'
+  get 'news_feed' => 'news_feed#index'
```

Comment:
Since it's a normal CRUD action, what about using rails resources and specify `only: [:index]` if that's the case?
https://stackoverflow.com/questions/13937670/rails-route-to-only-index